synfig-studio/src/gui/app.cpp

```diff
@@ -4389,12 +4390,12 @@ studio::App::setup_changed()
 void
 studio::App::process_all_events(long unsigned int us)
 {
-	Glib::usleep(us);
+	/*Glib::usleep(us);
```

Comment:
Just replace body with:
```
	while (gtk_events_pending()) {
		gtk_main_iteration_do(true);
	}
```

Reply:
I really don't know if it is even needed…

Reply:
> I really don't know if it is even needed…

Without this loop Splash window can't be rendered correctly, because we are doing all initialization work in the main GUI thread.

Reply:
So it was wrong. It should be done in a separate thread.
Should we fix it now?